tsconfig.json

```diff
@@ -1,30 +1,30 @@
-{
-  "extends": "./node_modules/@bentley/build-tools/tsconfig-base.json",
-  "compilerOptions": {
-    "skipLibCheck": true,
-    "baseUrl": "./node_modules",
-    "outDir": "./lib",
-    "resolveJsonModule": true,
-    "allowJs": true,
-    "esModuleInterop": true,
-    "jsx": "react",
-    "allowSyntheticDefaultImports": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "isolatedModules": true,
-    "noEmit": true,
-    "types": [
-      "@bentley/react-scripts"
-    ]
-  },
-  "include": [
-    "./src/**/*.ts",
-    "./src/**/*.tsx"
-  ],
-  "exclude": [
-    "build",
-    "lib",
-    "node_modules",
-    "./src/backend/**/*.ts"
-  ]
-}
+{
+  "extends": "./node_modules/@bentley/build-tools/tsconfig-base.json",
+  "compilerOptions": {
+    "skipLibCheck": true,
+    "baseUrl": "./node_modules",
+    "outDir": "./lib",
+    "resolveJsonModule": true,
+    "allowJs": true,
+    "esModuleInterop": true,
+    "jsx": "react-jsx",
```

Comment:
set this to jsx, and disable the new jsx-transform via env var. i think that should resolve the ci failure

Reply:
Thanks @aruniverse I learned that in the imodeljs repo and didn't apply it to this one yet.

Reply:
I'm assuming you were referring to `DISABLE_NEW_JSX_TRANSFORM=true`. If so, I'm still getting the same error.

Reply:
Just tested this locally and hitting an issue as well. However, it looks to be a problem with backend-webpack-tools rather than the react-scripts update. Digging now...

Reply:
The issue is related to the new version of webpack used for CRA 4.x. It's now using 4.44.2 rather than the version we're using everywhere else, 4.42.0. However, the new version is causing backend webpack tools to fail to build. Digging into it but I think it'll require an update in iModel.js.

Tempted to drop backend webpacking for the time being since we're facing other issues as well that still need to be investigated. Any issues with that?

Reply:
Ah, sorry didnt look closely enough and assumed you guys already were using `SKIP_PREFLIGHT_CHECK`, that looks right, its what we've had to do in other repos where cra uses a different version of webpack/jest/etc than whats defined as a dep/devDep.